hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/utils/AuditLogUtils.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.ozone.utils;
+
+import org.apache.hadoop.ozone.audit.AuditAction;
+import org.apache.hadoop.ozone.audit.AuditEventStatus;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Utility class to read audit logs.
+ */
+public final class AuditLogUtils {
```

Comment:
Since this is only for tests, I think `AuditLogTestUtils` would be better.  Also, moving it to `hdds-common` (e.g. in `hadoop-hdds/common/src/test/java/org/apache/hadoop/ozone/audit`) would allow more reuse.

For that `s3gateway` dependencies would need a new entry like:

https://github.com/apache/ozone/blob/b352ad03abd50ad8b2c74aa33173c2299afaf4c2/hadoop-ozone/ozone-manager/pom.xml#L106-L111

Reply:
I will update the PR